packages/components/table/src/util.ts

```diff
@@ -249,36 +249,31 @@ export function toggleRowStatus<T>(
   const index = statusArr.indexOf(row)
   const included = index !== -1
 
-  const addRow = () => {
-    statusArr.push(row)
-    if (isArray(row.children)) {
-      row.children.forEach((item) => {
-        toggleRowStatus(statusArr, item, newVal ?? !included)
-      })
+  const toggleStatus = (type: 'add' | 'remove'): boolean => {
```

Comment:
```suggestion
  const toggleStatus = (type: 'add' | 'remove') => {
```

This function should have no return value

Reply:
thank you for your patient correction